### PR TITLE
remove log4j from java cdk

### PR DESCRIPTION
**Pull request by stephane-airbyte**
## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

File: airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DateTimeConverter.kt [hunk 1]
```diff
@@ -5,17 +5,18 @@ package io.airbyte.cdk.db.jdbc
...
 import java.sql.*
 import java.time.*
 import java.time.format.DateTimeFormatter
 import java.util.concurrent.*
 import kotlin.math.abs
 import kotlin.math.min
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
+
+private val LOGGER = KotlinLogging.logger {}
```
Comment (edgao) [inline airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DateTimeConverter.kt:16]: curious: is it preferable to declare the logger outside the class?

File: airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DefaultJdbcDatabase.kt [hunk 2]
```diff
@@ -134,7 +132,5 @@ constructor(
             )
     }
 
-    companion object {
-        private val LOGGER: Logger = LoggerFactory.getLogger(DefaultJdbcDatabase::class.java)
-    }
+    companion object {}
```
Comment (edgao) [inline airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DefaultJdbcDatabase.kt:135]: nit: delete this?

**Review by edgao — APPROVED** [review]
rubberstamp :P

File: airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DateTimeConverter.kt (hunk 1 above)
Comment (gisripa) [reply airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DateTimeConverter.kt:16]: Yeah this is a question i have too. I'm inconsistently doing it inside the class and only outside when I need it in companion object.

Comment (stephane-airbyte) [reply airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/DateTimeConverter.kt:16]: I think it's either outside the class or inside the companion object. Definitely not inside the class, as that would create a new instance for each class instance

**Review by gisripa — APPROVED** [review]

**Comment by stephane-airbyte** [comment]
### Merge activity

* **May 23, 1:53 PM EDT**: @stephane-airbyte started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/airbytehq/airbyte/38583).

**Pull request merged**